ui/pages/Applications.tsx

```diff
@@ -40,13 +40,16 @@ function Applications({ className }: Props) {
     return () => clearInterval(interval);
   }, []);
 
+  const title = (
+    <Flex align start>
+      <h2 style={{ marginTop: "0px" }}>Applications</h2>
+      <Spacer padding="small" />
+      <PollingIndicator loading={loading} />
+    </Flex>
+  );
+
   return (
-    <Page className={className}>
-      <Flex align start>
-        <h2>Applications</h2>
-        <Spacer padding="xs" />
-        <PollingIndicator loading={loading} />
-      </Flex>
+    <Page className={className} title={title}>
```

Comment:
```suggestion
    <Page className={className}>
      {title}
```

Testing locally these two look the same now after your other styling changes. Could we just do that and skip the conditional logic in `Page.tsx` or is it useful to have for EE too?

Reply:
Refactored